### PR TITLE
output plots

### PR DESCRIPTION
**Pull request by davidslater**
Create perturbation-accuracy plots. Not sure the best place to have an example (test) results json.

### CODE REVIEW

**Comment by davidslater** [comment]
Example results json:
```
{
    "armory_version": "0.11.0-dev",
    "config": {
        "_description": "Resisc45 image classification, contributed by MITRE Corporation",
        "adhoc": null,
        "attack": {
            "knowledge": "white",
            "kwargs": {
                "batch_size": 1,
                "eps": 1,
                "eps_step": 0.125,
                "minimal": false,
                "num_random_init": 0,
                "targeted": false
            },
            "module": "armory.art_experimental.attacks.fgm_binary_search",
            "name": "FGMBinarySearch",
            "use_label": false
        },
        "dataset": {
            "batch_size": 16,
            "framework": "numpy",
            "module": "armory.data.datasets",
            "name": "resisc45"
        },
        "defense": null,
        "eval_id": "2020-07-17T203335.174651",
        "metric": {
            "means": true,
            "perturbation": "linf",
            "record_metric_per_sample": true,
            "task": [
                "categorical_accuracy"
            ]
        },
        "model": {
            "fit": false,
            "fit_kwargs": {},
            "model_kwargs": {},
            "module": "armory.baseline_models.keras.densenet121_resisc45",
           "name": "get_art_model",
            "weights_file": "densenet121_resisc45_v1.h5",
            "wrapper_kwargs": {}
        },
        "scenario": {
            "kwargs": {},
            "module": "armory.scenarios.image_classification",
            "name": "ImageClassificationTask"
        },
        "sysconfig": {
            "docker_image": "twosixarmory/tf1:0.11.0-dev",
            "external_github_repo": null,
            "gpus": "all",
            "output_dir": null,
            "output_filename": null,
            "use_gpu": false
        }
    },
    "results": {
        "adversarial_categorical_accuracy": [
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            1,
            0,
            0,
            0,
            0,
            1
        ],
        "adversarial_mean_categorical_accuracy": 0.0,
        "benign_categorical_accuracy": [
            1,
            0,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            1,
            0,
            1,
            1,
            1,
            1,
            1
        ],
        "benign_mean_categorical_accuracy": 1.0,
        "perturbation_linf": [
            0.5000000000000002,
            0.12500000000000022,
            0.12500000000000022,
            0.2500000000000002,
            0.12500000000000022,
            0.12500000000000022,
            0.12500000000000022,
            0.12500000000000022,
            0.12500000000000022,
            0.12500000000000022,
            0.12500000000000022,
            0.12500000000000022,
            0.12500000000000022,
            0.12500000000000022,
            0.12500000000000022,
            0.500000000000022
        ],
        "perturbation_mean_linf": 0.15625000000000017
    },
    "timestamp": 1595018060
}
```
